library/block/block_content_registry/src/testmod/resources/quilt.mod.json

```
@@ -0,0 +1,23 @@
+{
+  "schema_version": 1,
+  "quilt_loader": {
+    "group": "org.quiltmc.qsl.block",
+    "id": "qsl_block_content_registry_testmod",
```

```suggestion
    "id": "quilt_block_content_registry_testmod",
```
Which reminds me: i forgot to check the other PR for that